setuptools/config/_apply_pyprojecttoml.py

```diff
@@ -153,15 +153,17 @@ def _guess_content_type(file: str) -> Optional[str]:
     raise ValueError(f"Undefined content type for {file}, {msg}")
 
 
-def _long_description(dist: "Distribution", val: _DictOrStr, root_dir: StrPath):
+def _long_description(
+    dist: "Distribution", val: _LongDescriptionValue, root_dir: StrPath
+):
     from setuptools.config import expand
 
     if isinstance(val, str):
-        file: Union[str, list] = val
+        file: Union[str, tuple[()]] = val
```

Comment:
Curiosity question: what does this notation mean? `tuple of empty tuple`?

Reply:
It means an empty tuple `()`